std/encoding.d

```diff
@@ -2462,7 +2462,7 @@ abstract class EncodingScheme
      * Params:
      *    s = the array to be tested
      */
-    bool isValid(const(ubyte)[] s)
+    bool isValid()(const(ubyte)[] s)
```

Comment:
The overriding method must copy the base method's attributes because the compiler cannot check at runtime whether it's OK or not, and it doesn't know what the derived type is at compile time.

I'd say `@safe` and `nothrow` are easy to work around, but something like `pure` or `@nogc` would be difficult to deal with if you really needed to use global variables or the GC.

I'm not really sure what this means. In all likelihood, nobody has overridden these methods, so this breaking change won't break any actual code. But I don't have any experience deriving my own encoding type, so I don't know how people normally would use it.